packages/prisma/index.ts

```diff
@@ -23,6 +24,7 @@ export const customPrisma = (options?: Prisma.PrismaClientOptions) =>
   new PrismaClientWithoutExtension({ ...prismaOptions, ...options })
     .$extends(excludePendingPaymentsExtension())
     .$extends(bookingIdempotencyKeyExtension())
+    .$extends(undefinedGuardExtension())
```

Comment:
I feel like this needs a more descriptive name.

Reply:
Any suggestion?

Reply:
I renamed it `disallowUndefinedDeleteUpdateManyExtension`